functions-and-scope.js

```diff
@@ -50,14 +85,26 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomsten:
 // averageGrade(grades) geeft 6.642857142857143
-// averageGrade([6, 4, 5]) geeft xxxx
-// averageGrade([8, 9, 4, 6, 10]) geeft xxxx
+// averageGrade([6, 4, 5]) geeft 5
+// averageGrade([8, 9, 4, 6, 10]) geeft 7.4
+
+function averageGrade(gradesArray) {
+        let average = 0;
+
+        for (let i = 0; i < gradesArray.length; i++) {
+        average = average + gradesArray[i];
+
+        } return average / gradesArray.length;
+}
 
+console.log(averageGrade(grades))
 
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
+console.log(Math.round(averageGrade(grades)))
```

Comment:
Dit levert een heel getal op in plaats van afgerond op twee cijfers achter de komma. Je zou het volgende kunnen doen:
```suggestion
console.log(grades.toFixed(2))
```